schedule/yast/yast2_gui/yast2_gui_sle.yaml

```diff
@@ -10,21 +10,57 @@ vars:
     SOFTFAIL_BSC1063638: 1
     VALIDATE_ETC_HOSTS: 1
 schedule:
+    - "{{pre_boot_to_desktop}}"
     - boot/boot_to_desktop
     - console/prepare_test_data
     - console/consoletest_setup
     - console/hostname
-    - yast2_gui/yast2_control_center
-    - x11/yast2_lan_restart
-    - yast2_gui/yast2_bootloader
-    - yast2_gui/yast2_datetime
-    - yast2_gui/yast2_firewall
+    - x11/yast2_snapper
     - yast2_gui/yast2_hostnames
-    - yast2_gui/yast2_lang
-    - yast2_gui/yast2_network_settings
     - yast2_gui/yast2_software_management
     - yast2_gui/yast2_users
     - yast2_gui/yast2_security
     - yast2_gui/yast2_lan_restart_bridge
     - yast2_gui/yast2_lan_restart_vlan
-    - yast2_gui/yast2_lan_restart_bond
+    - "{{yast2_modules}}"
+conditional_schedule:
+    pre_boot_to_desktop:
+        ARCH:
+            aarch64:
+                - boot/uefi_bootmenu
+        BACKEND:
+            svirt:
+                - installation/bootloader_start
+    yast2_modules:
+        MACHINE:
```

Comment:
AFIK it would more fragile to set condition depending on machine (it could change in the future) it is better to use ARCH or BACKEND.

Reply:
Not possible in this case. Some modules fail for different machines in the same arch or backend.

Reply:
Normally we decide the machine at the upper level, job group yaml, for example, the test suite should not care about the machine IMO, it just a matter of us to make it work wherever we use it, so those conditions should be very general. You don't need to make it work for flavors which where not already existing, so yes, it could fail among different machines in the same arch but original fo the one that was working it should still working.

Reply:
Maybe I can leave it for now like this and later that we expend the coverage, we can change the conditional schedule, if conditions are still needed. So far, I can't figure out a way not to use "MACHINE". "ARCH" could almost do, but there is one module that works on "64bit" and doesn't work for other x86 machines.

Reply:
@rwx788 wdyt?

Reply:
I'm in general against conditional schedules. Also using machines here is more error prone as those might get renamed and we will get changed coverage without noticing it. AFAIR, if machine is used in job group, it's value is validated.

Reply:
I created separate schedule for xen hvm and used ARCH for conditional schedule.